tbx/core/blocks.py

```diff
@@ -614,7 +639,6 @@ class StoryBlock(blocks.StreamBlock):
     )
     embed = EmbedBlock(
         icon="code",
-        template="patterns/molecules/streamfield/blocks/embed_block.html",
         group="Media and images",
```

Comment:
Query would it make sense to move the icon and group attributes to the Meta class as well as the template definition?

Reply:
We could move the `icon`, however, since the `group` is defined on the StoryBlock for the other fields, I though it would be better to leave it here for consistency, and also to make it easy to update it if need arises, since one would be looking at all `group` definitions in one place.